libs/community/langchain_community/document_compressors/flashrank_rerank.py

```diff
@@ -26,8 +26,12 @@ class FlashrankRerank(BaseDocumentCompressor):
     """Flashrank client to use for compressing documents"""
     top_n: int = 3
     """Number of documents to return."""
+    score_threshold: float = 0.0
+    """Minimum relevance threshold to return."""
     model: Optional[str] = None
     """Model to use for reranking."""
+    prefix_metadata: str = ""
```

Comment:
Changed this from `Optional[str]` to `str` as there was no handling for the `None` case below (e.g., in `self.prefix_metadata + "id"`). Let me know if this wasn't intended.